### PR TITLE
Fix windows warnings

### PR DESCRIPTION
**Pull request by nkoenig**

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

**Review by ahcorde — CHANGES_REQUESTED** [review]
CI is failing

**Comment by azeey** [comment]
@nkoenig to get back to this

**Comment by mjcarroll** [comment]
@osrf-jenkins retest this please

**Comment by nkoenig** [comment]
This doesn't fix all of the windows warnings, but it has reduced them. I think it's okay to get this in.

File: plugins/joy_to_twist/JoyToTwist.cc [hunk 1]
```diff
@@ -49,6 +49,22 @@ void setVectorFromString(const std::string &_str,
   }
 }
 
+//////////////////////////////////////////////////
+// String to vector helper function.
+void setVectorFromString(const std::string &_str,
```
Comment (ahcorde) [inline plugins/joy_to_twist/JoyToTwist.cc:54]: include `<string>`

**Review by ahcorde — APPROVED** [review]

**Pull request merged**